test/e2e/mock-cdn/cdn-stale-diff-res-headers.json

```diff
@@ -1,3 +1,4 @@
 {
-  "Etag": "W/\"ece7f5f533b8978063633ea5b1f8a0fc\""
+  "Content-Type": "text/plain",
```

Comment:
with this addition, we are now able to see the mocked response in the network tab (see attached video in the description)